python/amici/sbml_import.py

```diff
@@ -473,15 +475,26 @@ def _gather_locals(self) -> None:
         for r in self.sbml.getListOfRules():
             self.local_symbols[r.getVariable()] = sp.Symbol(r.getVariable(),
                                                             real=True)
-
-        for r in self.sbml.getListOfReactions():
-            if r.isSetId():
-                self.local_symbols[r.getId()] = _get_identifier_symbol(r)
-
         # SBML time symbol + constants
         self.local_symbols['time'] = sp.Symbol('time', real=True)
         self.local_symbols['avogadro'] = sp.Symbol('avogadro', real=True)
 
+        # definitions below already use self.local_symbols and shouldn't be
+        # reordered
+        for r in self.sbml.getListOfReactions():
+            for e in list(r.getListOfReactants()) + \
+                     list(r.getListOfProducts()):
```

Comment:
```suggestion
            for e in chain(r.getListOfReactants(), r.getListOfProducts()):
```
`from itertools import chain`
?

Reply:
👍